src/go/rpk/pkg/config/schema.go

```diff
@@ -64,8 +64,9 @@ type Pandaproxy struct {
 }
 
 type SchemaRegistry struct {
-	SchemaRegistryAPI    []NamedSocketAddress `yaml:"schema_registry_api,omitempty" mapstructure:"schema_registry_api,omitempty" json:"schemaRegistryApi,omitempty"`
-	SchemaRegistryAPITLS []ServerTLS          `yaml:"schema_registry_api_tls,omitempty" mapstructure:"schema_registry_api_tls,omitempty" json:"schemaRegistryApiTls,omitempty"`
+	SchemaRegistryAPI               []NamedSocketAddress `yaml:"schema_registry_api,omitempty" mapstructure:"schema_registry_api,omitempty" json:"schemaRegistryApi,omitempty"`
+	SchemaRegistryAPITLS            []ServerTLS          `yaml:"schema_registry_api_tls,omitempty" mapstructure:"schema_registry_api_tls,omitempty" json:"schemaRegistryApiTls,omitempty"`
+	SchemaRegistryReplicationFactor *int                 `yaml:"schema_registry_replication_factor,omitempty" mapstructure:"schema_registry_replication_factor,omitempty" json:"schemaRegistryReplicationFactor,omitempty"`
```

Comment:
Is our goal to remove the schema registry config from this configuration entirely at a future date (similar to removing the `redpanda:` config as well)?

This is consistent with the prior fields, currently. We do not need the `schema_registry_` prefix in the tags, because this is already within the schema registry struct, but for consistency we may as well keep it for the time being.

Reply:
I wasn't thinking about anything specific, I just don't know enough about rpk to review reliably. maybe upgrade issues lurking?